### ISSUE TITLE
3.1.2 Enterprise - Security Error - Token Create Operations Are Not Allowed

### ISSUE THREAD

**Issue by tuddman** [neo4j-contrib/spatial#303]
not sure if this is the right place to post this...   (if this is instead a neo4j Enterprise issue?)

but on a freshly installed instance of `neo4j-enterprise-3.1.2`, with `spatial-0.24-3.1.1` installed into `/plugins`  and  having logged in as (default) user `neo4j`   trying to `call`:

```
CALL spatial.addPointLayer('locations')
```

results in:
```
Token create operations are not allowed for user 'neo4j' with roles [admin] restricted to WRITE.
```

I haven't altered any of the roles, and as its the `admin` role with privileges described [here](https://neo4j.com/docs/operations-manual/current/security/authentication-authorization/native-user-role-management/native-roles/)   not sure what I'm doing wrong or missing.

following the exact same setup procedures, but with `neo4j-community-3.1.2`   works fine.  `ReferenceNode` label(s) were created, GeoSpatial nodes can be added, etc.

worth noting?:  other `apoc` procedures worked fine in the `enterprise` instance.

**Comment by tuddman** [comment]
looks like the `apoc` calls were successful because of [this commit](https://github.com/neo4j-contrib/neo4j-apoc-procedures/commit/3d808488f877b4d36155ac094799c6561a739814)

**Comment by craigtaverner** [comment]
You are correct. Neo4j enterprise 3.1.2 tightened security in a non-compatible way. We could release spatial for 3.1.2 with the same fix that APOC used, or we could wait for 3.1.3 which is likely (but not yet certain) to reverse the change. Do you need spatial on 3.1.2 urgently? I suspect 3.1.3 will not be long.

**Comment by tuddman** [comment]
no, not urgent to get spatial on 3.1.2 enterprise for any specific reason(s) right now.
just trying it out on the latest.    can use community edition for now.
though will appreciate it to function properly on _some_ recent enterprise release in the coming weeks.
if 3.1.3 doesn't reverse course (and fix) then we can follow path that APOC took. Sound good?

**Comment by tuddman** [comment]
@craigtaverner thanks!

with [#9058](https://github.com/neo4j/neo4j/pull/9058) and release of `3.1.3`
 haven't fully tested all other functionality, but:  confirmed fix for this issue is to upgrade to `neo-3.1.3-enterprise`
